hyp3_gamma/insar/getDemFileGamma.py

```diff
@@ -5,9 +5,9 @@
 from hyp3_gamma.dem import get_geometry_from_kml, prepare_dem_geotiff
 
 
-def get_dem_file_gamma(dem_image: str, dem_par: str, safe_dir: str, pixel_size: int):
+def get_dem_file_gamma(dem_image: str, dem_par: str, safe_dir: str, pixel_size: int, water_masking: bool):
```

Comment:
`water_masking` parameter doesn't seem to be used here?

Reply:
we do not use water-masking to the dem.tif, instead, we use water-masking in the next right before we output the geotiff file. It turns to be good results. So there is no need to input the water-masking argument.

Keep it as it is.

Reply:
So you should remove this water masking argument to this call

Reply:
Yes. I delete the water_masking. Sorry to the confusing.